API.md

```diff
@@ -50,12 +61,34 @@ Expose the chordRendererFactory_new functionmodule_"></a>
 
 ## chordRendererFactory
 Expose the chordRendererFactory() function
+<a name="defaultAltIntervals"></a>
+
+## defaultAltIntervals : [<code>AltIntervals</code>](#AltIntervals)
+Default alterations triggered by the use of the alt modifier, eg none.
```

Comment:
shouldn't it be `eg all` here?
By default, you get all alterations isn't it?

Reply:
I decided to go the other way around actually. I'd like to hear more user feedback before enforcing something by default. So for now if you want everything, you just need to configure you parser instance to your taste.